plugins/nginx-vhosts/commands

```diff
@@ -180,18 +28,52 @@ EOF
     fi
 
     tail "$NGINX_LOGS_ARGS" "$NGINX_LOGS_PATH"
-    ;;
+  ;;
+
+  nginx:disable)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    verify_app_name "$2"
+    APP="$2"
+
+    eval "$(config_export app $APP)"
+    if [[ -z "$NO_VHOST" ]]; then
+      config_set --no-restart $APP NO_NGINX=1
+      config_set $APP NO_VHOST=1
+    elif [[ -z "$NO_NGINX" ]]; then
+      config_set --no-restart $APP NO_NGINX=1
+      nginx_build_config $APP
+    else
+      dokku_log_info1 "nginx is already disable for app ($APP)"
+    fi
+  ;;
+
+  nginx:enable)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    verify_app_name "$2"
+    APP="$2"
+
+    eval "$(config_export app $APP)"
+    if [[ -n "$NO_NGINX" ]]; then
+      config_unset --no-restart $APP NO_NGINX
+      unset NO_NGINX
+      nginx_build_config $APP
+    else
+      dokku_log_info1 "nginx is already enabled for app ($APP)"
+    fi
+  ;;
 
   help | nginx:help)
     cat<<EOF
     nginx:build-config <app>, (Re)builds nginx config for given app
     nginx:access-logs <app> [-t], Show the nginx access logs for an application (-t follows)
     nginx:error-logs <app> [-t], Show the nginx error logs for an application (-t follows)
+    nginx:disable <app>, disable nginx for an application (forces container binding to external interface)
+    nginx:enable <app>, enable nginx for an application
```

Comment:
Heh, re: refactoring how we trigger plugin commands, would be cool to see these become `declare=` calls in individual functions somehow. Or at least move the declaration of the help output from a command to where the command is declared.